examples/client.nim

```diff
@@ -1,8 +1,9 @@
-import ../src/ws, nativesockets, chronos, os, chronicles, stew/byteutils
+import ../src/ws, nativesockets, chronos,chronicles, stew/byteutils
 
 proc main() {.async.} =
-  let ws = await connect(
-    "127.0.0.1", Port(8888),
+  let ws = await wsConnect(
+    "127.0.0.1", 
+    Port(8888),
```

Comment:
I think defining `connect` as `proc connect(_: type WebSocket, ...)` is more nim'ish and it allows you to use it as a static proc - `WebSocket.connect(...)`